state/filesystem.go

```diff
@@ -456,6 +477,16 @@ func (sb *storageBackend) MachineFilesystemAttachments(machine names.MachineTag)
 	return attachments, nil
 }
 
+// UnitFilesystemAttachments returns all of the FilesystemAttachments for the
+// specified unit.
+func (sb *storageBackend) UnitFilesystemAttachments(unit names.UnitTag) ([]FilesystemAttachment, error) {
+	attachments, err := sb.filesystemAttachments(bson.D{{"hostid", unit.Id()}})
+	if err != nil {
+		return nil, errors.Annotatef(err, "getting filesystem attachments for unit %q", unit.Id())
```

Comment:
Use ReadableString here for consistency?

Reply:
Done, although we typically use ReadableString() for cases where the tag kind is unknown